can-stache-bindings.js

```diff
@@ -519,6 +524,21 @@ var behaviors = {
 			event = eventBindingData.eventName;
 			bindingContext = eventBindingData.bindingContext;
 			bindingContextObservable = eventBindingData.bindingContextObservable;
+
+			//!steal-remove-start
+			if(process.env.NODE_ENV !== "production") {
+				if(
+					!eventBindingData.bindingCode &&
+					el[canSymbol.for("can.viewModel")] &&
+					("on" + event) in HTMLElement.prototype
```

Comment:
Need to use `GLOBAL().HTMLElement.prototype` in order to avoid SSR issues.

Reply:
Can this be `GLOBAL().Element`?  HTMLElement is not available from can-vdom, but Element is.

Reply:
Element.prototype has very few events (8) in comparison to HTMLElement.prototype (75 more), so it's not a substitute, but it's a good idea to test for its existence (and use GLOBAL()) before reading into it.  It isn't necessary to warn in SSR; it will still warn in the browser console when hydrated.

Reply:
A better idea might even be `("on" + event) in el` which removes the need to check for a global

Reply:
👍